doc/source/examples.rst

```diff
@@ -0,0 +1,149 @@
+.. _examples:
+
+Examples
+########
+
+Welcome! This guide offers several examples to help you effectively utilize this package.
+
+Files needed:
+
+    1. :download:`Ni-xray.gr <examples/Ni-xray.gr>` - experimental X-ray PDF data
+    2. :download:`Ni.stru <examples/Ni.stru>` - Ni f.c.c. structure in PDFfit format
+
+======================================
+Example 1: Calculate PDF of FCC nickel
+======================================
+
+The first example shows how to calculates the PDF for FCC nickel and saves the resulting data to a file and plot it using matplotlib.
+
+1. Imports the PdfFit class from the diffpy.pdffit2 module::
+
+    from diffpy.pdffit2 import PdfFit
+
+2. Create a PDF calculator object and assigned to the variable ``P``. Make sure the ``Ni.stru`` file is in the same directory as the script and you've cd to the directory, load structure file. Then allocate and configure PDF calculation and run the calculation::
+
+    # create new PDF calculator object
+    P = PdfFit()
+
+    # load structure file in PDFFIT or DISCUS format
+    P.read_struct("Ni.stru")
+
+    radiation_type = "X"  # x-rays
+    qmax = 30.0  # Q-cutoff used in PDF calculation in 1/A
+    qdamp = 0.01  # instrument Q-resolution factor, responsible for PDF decay
+    rmin = 0.01  # minimum r-value
+    rmax = 30.0  # maximum r-value
+    npts = 3000  # number of points in the r-grid
+
+    # allocate and configure PDF calculation
+    P.alloc(radiation_type, qmax, qdamp, rmin, rmax, npts)
+    P.calc()
+
+3. Save the refined result::
+
+    P.save_pdf(1, "Ni_calculation.cgr")
+
+4. We can also plot it using matplotlib::
+
+    import matplotlib.pyplot as plt
+
+    # obtain list of r-points and corresponding G values
+    r = P.getR()
+    G = P.getpdf_fit()
+
+    plt.plot(r, G)
+    plt.xlabel("r (Å)")
+    plt.ylabel("G (Å$^{-2}$)")
+    plt.title("x-ray PDF of nickel simulated at Qmax = %g" % qmax)
+
+    # display plot window, this must be the last command in the script
+    plt.show()
+
+The scripts can be downloaded :download:`here <examples/Ni_calculation.py>`. 
+
+=======================================
+Example 2: Performing simple refinement
+=======================================
+
+The second example shows how to perform simple refinement of Ni structure to the experimental x-ray PDF. The example uses the same data files as the first example.
+
+1. Imports the PdfFit class from the diffpy.pdffit2 module::
+
+    from diffpy.pdffit2 import PdfFit
+
+2. Load experimental x-ray PDF data and nickel structure file::
+
+    # Load experimental x-ray PDF data
+    qmax = 30.0  # Q-cutoff used in PDF calculation in 1/A
+    qdamp = 0.01  # instrument Q-resolution factor, responsible for PDF decay
+    pf.read_data("Ni-xray.gr", "X", qmax, qdamp)
```

Comment:
where is pf defined?  I don't think this code will work will it?

The best way to write these docs is to copy-paste the code from the working examples I would think.  Otherwise it is way too hard to maintain.

Reply:
yeah I somehow forgot to copy that line... It should be fixed now.